packages/cpse-template/src/intercept.js

```diff
@@ -14,6 +14,6 @@ module.exports = targets => {
          *  Wee need to activated esModules and cssModules to allow build pack to load our extension
          * {@link https://magento.github.io/pwa-studio/pwa-buildpack/reference/configure-webpack/#special-flags}.
          */
-        flags[targets.name] = {esModules: true, cssModules: true};
+        flags[targets.name] = { esModules: true, cssModules: true, graphqlQueries: true };
```

Comment:
This flag leads to performance impact for all modules that don't want to induce own queries. 
I will write a FAQ page where I documented this issue and how to solve it. 

Thank you for your contribution.